src/loan-options/index.html

```diff
@@ -146,8 +146,8 @@ <h3 class="section-heading">What to know</h3>
                       </ol>
 
                       <p>Rates vary among lenders, especially for shorter terms.
-                        <a href="/owning-a-home/check-rates" target="_blank">Explore rates specific to your situation</a> so you can tell if you're getting a good deal.
-                         Always compare official loan offers, called <a href="/askcfpb/146/what-is-a-good-faith-estimate-what-is-a-gfe.html" id="good-faith-link" target="_blank">Good Faith Estimates</a>, before making your decision.</p>
+                        <a href="/owning-a-home/check-rates" target="_blank">Explore rates for different loan terms</a> so you can tell if you're getting a good deal.
+                         Always compare official loan offers, called <a href="http://consumerfinance.gov/askcfpb/1995/what-is-a-loan-estimate.html" target="_blank">Loan Estimates</a>, before making your decision.</p>
```

Comment:
Looks like you will have to update browser tests for links on this page too https://github.com/cfpb/owning-a-home/blob/master/test/browser_testing/features/loan_options.feature